Makefile

```diff
@@ -682,7 +682,6 @@ $(addprefix .build-artefacts/annotated/, $(SRC_JS_FILES) src/TemplateCacheModule
 ${PYTHON_VENV}:
 	mkdir -p .build-artefacts
 	virtualenv --no-site-packages $@
-	${PIP_CMD} install -U pip
```

Comment:
It was for upgrading pip. But seems a bad idea because it breaks the teste2e. Don't know why.